chainerx_cc/chainerx/python/routines.cc

```diff
@@ -547,6 +547,7 @@ void InitChainerxManipulation(pybind11::module& m) {
           [](const ArrayBodyPtr& a, int8_t axes) { return MoveArrayBody(Transpose(Array{a}, {axes})); },
           "a"_a,
           "axes"_a = nullptr);
+    m.def("flatten", [](const ArrayBodyPtr& a) { return MoveArrayBody(Flatten(Array{a})); }, "a"_a);
```

Comment:
Move the definition to `chainerx_cc/chainerx/python/array.cc`.

Reply:
Done.